### PR TITLE
Improve manifesto page and add twitter share button

### PR DESCRIPTION
**Pull request by luisiul359**
@jagoncalves14 please review this.

Steps to populate DB correctly:

1. Add this to `db/seeds.rb`

```
AdminUser.delete_all	
ManifestoItem.delete_all	
ManifestoSection.where.not(manifesto_section_id: nil).delete_all	
ManifestoSection.delete_all	
Manifesto.delete_all	
Candidate.delete_all
Party.delete_all	
```

2. drop all entries in DB running `bundle exec rake db:seed`
3. populate DB running `bundle exec rake data:populate`

Some errors to fix:

1. We need to click on Twitter logo to share, clicking in the black part does not work

image

2. Sections menu is not responsive:

image

3. Sections menu disappears when window is too small (or mobile)

image

4. Decide what to do regarding section's name too long

### CODE REVIEW

File: app/javascript/components/party/party-manifestos/index.js [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+                                    selectedKey={this.getSelectedKey(sections, section_id)}
+                                    openKey={this.getOpenKey(sections, section_id)}
+                                />
+                            )}
+                        </Sider>
+                        <Layout.Content>
+                            <ManifestoSection title={title} section={section} section_id={section_id} />
+                        </Layout.Content>
+                    </Layout>
+                    <LayoutFooter />
```
Comment (luisiul359) [inline app/javascript/components/party/party-manifestos/index.js]: @nancruz @jagoncalves14 I think this should be below `</Layout.Content>` but only works here..

**Comment by luisiul359** [comment]
Iv'e added all these bugs in #159

File: app/assets/stylesheets/pages/home/_home-videos.scss [hunk 2]
```diff
@@ -17,23 +17,19 @@ section.home-videos {
...
 
     .home-videos-player-wrapper {
-      position: relative;
-      padding-top: 40%;
-
-      @media only screen and (max-width: 768px) {
-        padding-top: 60%
-      }
+        position: relative;
+        padding-top: 59%;
```
Comment (luisiul359) [inline app/assets/stylesheets/pages/home/_home-videos.scss:27]: Isn't this too big in desktop?

File: app/javascript/components/party/party-manifestos/index.js (hunk 1 above)
Comment (jagoncalves14) [reply app/javascript/components/party/party-manifestos/index.js]: Fixed while being inside <Layout>

File: app/assets/stylesheets/pages/home/_home-videos.scss (hunk 2 above)
Comment (jagoncalves14) [reply app/assets/stylesheets/pages/home/_home-videos.scss:27]: This is how it shows up to me:
https://d.pr/i/UFpP2S

**Comment by jagoncalves14** [comment]
This PR fixes issues:
1. Twitter logo click;
2. Sections menu is now responsive;
3. Footer positioning issue;

**Review by nancruz — APPROVED** [review]

**Pull request merged**